sentry-log/src/logger.rs

```diff
@@ -4,14 +4,9 @@ use crate::converters::{breadcrumb_from_record, event_from_record};
 use crate::integration::with_integration;
 
 /// Provides a dispatching logger.
+#[derive(Default)]
 pub struct Logger;
 
-impl Logger {
-    pub fn new() -> Self {
```

Comment:
TBH, having `new` even on types implement `Default` is not a bad idea. See `Vec` or `HashMap`.

Reply:
👍 I wrote that down on my list of additional cleanup tasks that I want to do after all the refactorings